### PR TITLE
Adds a getter for a bulk op's write concern

### PR DESCRIPTION
**Pull request by mschoenlaub**
Fullfills https://jira.mongodb.org/browse/CDRIVER-488

### CODE REVIEW

File: src/mongoc/mongoc-bulk-operation.c [hunk 1]
```diff
@@ -370,6 +370,13 @@ mongoc_bulk_operation_set_write_concern (mongoc_bulk_operation_t      *bulk,
    }
 }
 
+const mongoc_write_concern_t *
+mongoc_bulk_operation_get_write_concern (mongoc_bulk_operation_t      *bulk)
```
Comment (TylerBrock) [inline src/mongoc/mongoc-bulk-operation.c]: I don't think we need the extra whitespace here between mongoc_bulk_operation_t and *bulk.

**Comment by TylerBrock** [comment]
Hey @mschoenlaub thanks for the pull request. It looks ok to me, maybe @hanumantmk can take a look as well.

**Comment by mschoenlaub** [comment]
Hey @TylerBrock , unnecessary whitespace has been removed.

File: src/mongoc/mongoc-bulk-operation.c [hunk 2]
```diff
@@ -370,6 +370,13 @@ mongoc_bulk_operation_set_write_concern (mongoc_bulk_operation_t      *bulk,
    }
 }
 
+const mongoc_write_concern_t *
+mongoc_bulk_operation_get_write_concern (mongoc_bulk_operation_t *bulk)
```
Comment (mschoenlaub) [inline src/mongoc/mongoc-bulk-operation.c]: Hey @TylerBrock - unncessary whitespace has been removed.

File: src/mongoc/mongoc-bulk-operation.c [hunk 3]
```diff
@@ -370,6 +370,13 @@ mongoc_bulk_operation_set_write_concern (mongoc_bulk_operation_t      *bulk,
...
+{
+   bson_return_if_fail (bulk);
```
Comment (TylerBrock) [inline src/mongoc/mongoc-bulk-operation.c]: This should be `bson_return_val_if_fail (bulk, NULL);`

File: src/mongoc/mongoc-bulk-operation.h [hunk 4]
```diff
@@ -76,7 +76,7 @@ void                     mongoc_bulk_operation_set_client        (mongoc_bulk_op
                                                                   void                         *client);
 void                     mongoc_bulk_operation_set_hint          (mongoc_bulk_operation_t      *bulk,
                                                                   uint32_t                      hint);
```
Comment (TylerBrock) [inline src/mongoc/mongoc-bulk-operation.h]: Sorry for the ASCII art changes but can we line up the above function parameters with those of mongoc_bulk_operation_get_write_concern's below?

**Comment by TylerBrock** [comment]
Thanks, this look good.

File: src/mongoc/mongoc-bulk-operation.h [hunk 5]
```diff
@@ -65,18 +65,18 @@ void mongoc_bulk_operation_update_one  (mongoc_bulk_operation_t       *bulk,
...
+                                                                       const mongoc_write_concern_t *write_concern);
+void                          mongoc_bulk_operation_set_database      (mongoc_bulk_operation_t      *bulk,
+                                                                       const char                   *database);
+void                          mongoc_bulk_operation_set_collection    (mongoc_bulk_operation_t      *bulk,
+                                                                       const char                   *collection);
+void                          mongoc_bulk_operation_set_client        (mongoc_bulk_operation_t      *bulk,
+                                                                       void                         *client);
+void                          mongoc_bulk_operation_set_hint          (mongoc_bulk_operation_t      *bulk,
+                                                                       uint32_t                      hint);
+const mongoc_write_concern_t *mongoc_bulk_operation_get_write_concern (mongoc_bulk_operation_t      *bulk);
```
Comment (hanumantmk) [inline src/mongoc/mongoc-bulk-operation.h]: Is it possible for the mongoc_bulk_operation_t* to be const here?

**Pull request closed without merging**